src/Plugin/Deriver/ContactsEntityBlockDeriver.php

```diff
@@ -54,7 +54,7 @@ public static function create(ContainerInterface $container, $base_plugin_id) {
   public function getDerivativeDefinitions($base_plugin_definition) {
     foreach ($this->entityTypeManager->getDefinitions() as $entity_type_id => $entity_type) {
       // We are only interested in entity types that have been approved.
-      if (empty($entity_type->contacts_entity)) {
+      if (empty($entity_type->get('contacts_entity'))) {
```

Comment:
We don't need `empty()` for this and we know the value will either be NULL or a bool, so `if ($entity_type->get('contacts_entity')) {` should be sufficient.